libs/designsystem/src/lib/scss/base/_lists.scss

```diff
@@ -0,0 +1,89 @@
+$bullet-font-size: 1.3em;
+$hollow-bullet-font-size: 0.8em;
+$list-item-line-height: 1.2em;
+
+// TODO: Move to utilities/functions
+@function unitless($number) {
+  @return $number / ($number * 0 + 1);
+}
+
+// -----------------------------------------------------------------------------
+// Grid layout
+// -----------------------------------------------------------------------------
+li {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  // Nested lists
+  ol,
+  ul {
+    grid-column: 2 / -1;
+  }
+}
+
+// -----------------------------------------------------------------------------
+// Spacing
+// -----------------------------------------------------------------------------
+ol,
+ul {
+  padding-inline-start: size('xxxs'); // left
+}
+
+li {
+  &::before {
+    padding-inline-end: size('xs'); // right
+  }
+
+  // Vertical space between list items (excluding the first outermost item)
+  li,
+  & + li {
+    margin-block-start: size('xxs'); // top
+  }
+}
+
+// -----------------------------------------------------------------------------
+// Markers (bullets and numbers)
+// -----------------------------------------------------------------------------
```

Comment:
In the mixed unordered/ordered list example third level are displayed with circle markers:

![image](https://user-images.githubusercontent.com/150451/134644677-6645b30e-439f-4edc-af29-b70dbcf52392.png)

They should be squares, so the selectors below for second and third level markers need to be extended, so they match even if unordered lists are mixed with ordered lists.

Reach out to me if I'm not making sense 😉

Reply:
I'll add suggestions below 👇